modules/va_forms/app/sidekiq/va_forms/form_builder.rb

```diff
@@ -6,7 +6,6 @@
 module VAForms
   class FormBuilder
     include Sidekiq::Job
-    include SentryLogging
```

Comment:
We're no longer logging to Sentry in this job, and all references were already removed.

Reply:
![cleaning-gif-by-spongebob-squarepants](https://github.com/department-of-veterans-affairs/vets-api/assets/8226466/b6cb55e2-29ea-4f10-831f-82602d6203aa)